src/chains/erc20/Listener.ts

```diff
@@ -140,6 +134,7 @@ export class Listener extends BaseListener<
 
     // process events in sequence
     for (const e of cwEvents) {
+      // TODO: is this correct?
```

Comment:
This is indeed correct as just passing 'erc20' instead of the actual token name would be like passing 'substrate' instead of the actual substrate chain name in the substrate listener. Shouldn't cause any issues.

Reply:
Okay, in that case, when I run `yarn listen` it all says `erc20` as chain instead of whatever the token name is, so I need to fix something else somewhere.

How can I test this code block? In theory the `never` type should throw an error here. The entire "supported chains" thing could use an overhaul.